File: CompositeNumber.py
num = int(input("Enter a number: "))

if num <= 1:
    print("Not a composite number")
else:
    composite = False
    for i in range(2, int(num**0.5) + 1):
        if num % i == 0:
            composite = True
            break

    if composite:
        print(num, "is a Composite number")
    else:
        print(num, "is NOT a Composite number")

File: CompositeNumber.java
import java.util.Scanner;

public class CompositeNumber {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int num = sc.nextInt();

        if (num <= 1) {
            System.out.println("Not a composite number");
            return;
        }

        boolean composite = false;

        for (int i = 2; i <= Math.sqrt(num); i++) {
            if (num % i == 0) {
                composite = true;
                break;
            }
        }

        if (composite)
            System.out.println(num + " is a Composite number");
        else
            System.out.println(num + " is NOT a Composite number");
    }
}
